homework-submission/react-hyf/react-hyf-client/src/components/UsersList/UsersList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './UsersList.css';
+import UsersAvatar from '../UsersAvatar/UsersAvatar';
+import UsersField from '../UsersField/UsersField';
+
+const UsersList = ({ data }) => (
+  <div className="users">
+    {data.map((user, index) => (
+      <div key={index} className="avatar-box">
+        <UsersAvatar
+          name={user.name}
+          url={'https://uinames.com/api/photos/'.concat(user.gender, '/', index + 1, '.jpg')}
```

Comment:
You can also make use of backticks which allows you to write variables inside the string and write it like this: 

```
 url={`https://uinames.com/api/photos/${user.gender}/${index + 1}.jpg`}
```